Fix issue when generating root fs

jockebech opened the pull request:
The line getting all folders for the file list generation also picked up
the "." folder, which incorrectly generated in a line saying
  "dir  755 0 0"
which doesn't make sense to cpio and hence failed.

Signed-off-by: Joakim Bech joakim.bech@linaro.org

jbech-linaro commented:
Even though untested, this is a good start, I'll merge it.

vchong commented:
Thanks! I was able to build all the images today but the kernel crashed while booting. Will try to debug and PR a fix.

vchong commented:
Fixed in PR #2.

jockebech committed:
Fix issue when generating root fs

The line getting all folders for the file list generation also picked up
the "." folder, which incorrectly generated in a line saying
  "dir  755 0 0"
which doesn't make sense to cpio and hence failed.

Signed-off-by: Joakim Bech <joakim.bech@linaro.org>
